### PR TITLE
[3.3.5] Core/Creature: add Rend to kaelthas

### PR DESCRIPTION
**Pull request by w5860363**
Add Rend（http://www.wowhead.com/spell=36965） 
close https://github.com/TrinityCore/TrinityCore/issues/15915
http://wowwiki.wikia.com/wiki/Thaladred_the_Darkener

### CODE REVIEW

**Comment by Aokromes** [comment]
Plz. don't close older PRs when you update them, simply update them without close old pr.

**Comment by w5860363** [comment]
@Aokromes receive

**Comment by Rushor** [comment]
@w5860363 : simply add new commits to the pr and then use: https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

File: src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp [hunk 1]
```diff
@@ -939,6 +942,15 @@ class boss_thaladred_the_darkener : public CreatureScript
                 else
                     Silence_Timer -= diff;
 
+                //Rend_Timer
+                 if (Rend_Timer <= diff)
+                 {
+                    DoCastVictim(SPELL_REND);
+                    Rend_Timer = 8000;
```
Comment (Nyeriah) [inline src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp]: Sniffs show it being cast at 4 seconds always. But don't change it now, leave it as it is.

Comment (w5860363) [inline src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp]: @Nyeriah if Rend_Timer = urand(4000, 8000); Can you solve this？

Comment (Nyeriah) [inline src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp]: On sniffs they're always cast at 4 seconds, I have solved it in the commit

**Pull request closed without merging**